server/templates/details.html.tmpl

```diff
@@ -213,6 +221,11 @@
 
 {{define "result-reviews-count"}}This rule requires at least {{.Count}} approval{{if gt .Count 1}}s{{end}}{{end}}
 
+{{define "result-conditions-details"}}
+  {{/* TODO(bkeyes): this is a placeholder until I can refactor predicate rendering */}}
```

Comment:
If this something you want to do in a follow up PR?

Reply:
Yeah, trying to refactor the predicate display in this PR was going to overwhelm the actual changes.